src/Converter.cpp

```diff
@@ -213,21 +277,22 @@ Converter::~Converter()
     assert(!data);
```

Comment:
Can't compile this:

```
In file included from /usr/include/c++/9/cassert:44,
                 from /home/afabisch/Projekte/pocolog2msgpack_test/tools/pocolog2msgpack/src/Converter.cpp:14:
/home/afabisch/Projekte/pocolog2msgpack_test/tools/pocolog2msgpack/src/Converter.cpp: In destructor ‘virtual Converter::~Converter()’:
/home/afabisch/Projekte/pocolog2msgpack_test/tools/pocolog2msgpack/src/Converter.cpp:277:13: error: ‘data’ was not declared in this scope
  277 |     assert(!data);
```

I guess you build with `-DNDEBUG` (release mode?). Delete this line if you don't need the `data` anymore.